proto/cosmos/tx/signing/v1beta1/signing.proto

```diff
@@ -52,7 +52,12 @@ enum SignMode {
   // EIP-191 in the future.
   //
   // Since: cosmos-sdk 0.45.2
-  SIGN_MODE_EIP_191 = 191;
+  // Deprecated post 0.47.x: Sign mode refers to a method of encoding string data for
```

Comment:
Please instead use "Deprecated: Post 0.47.X in which Sign mode..." as "Deprecated: " will allow Go editors and tools to pick up that generated Go code being used is deprecated https://rakyll.org/deprecated/